apps/minifront/src/fetchers/staking.ts

```diff
@@ -1,93 +0,0 @@
-import {
```

Comment:
All of this is now moved into the `ViewService#delegationsByAddressIndex` method impl